iml-services/iml-ntp/src/main.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+#![feature(async_await)]
+
+mod error;
+
+use error::NtpError;
+use futures::{
+    compat::{Future01CompatExt, Stream01CompatExt},
+    future::{FutureExt, TryFuture, TryFutureExt},
+    stream::{Stream, StreamExt, TryStreamExt},
+};
+use iml_postgres::connect;
+use iml_rabbit::connect_to_rabbit;
+use iml_service_queue::service_queue::{consume_service_queue, data_only, into_deserialized};
+use iml_wire_types::{
+    ntp::{TimeStatus, TimeSync},
+    Fqdn,
+};
+use std::io;
+use tokio_postgres::Client;
+use tracing_subscriber::{fmt::Subscriber, EnvFilter};
+
+/// This async function will retrieve the content type id of the specified model. A future
+/// containing the content type id of the model is returned.
+async fn get_managed_host_content_type_id(client: &mut Client) -> Result<i32, NtpError> {
+    client
+        .prepare("SELECT id FROM django_content_type WHERE model='managedhost';")
```

Comment:
Should create a [`Name` trait](https://docs.rs/iml-wire-types/0.2.0/iml_wire_types/db/trait.Name.html) for this in wire-types.

Reply:
Looks like I can get rid of this function after a little more research.